### PR TITLE
Update bootstrap from V4-Alpha6 to V4.3.1

### PR DESCRIPTION
**Pull request by ShmoogleOsukami**
This is the absolute minimum amount of work required to get bootstrap updated.

There is a lot of classes that'll need updating but as of right now everything looks and functions the same as before.

This will be a good foundation for getting everything cleaned up. We'll be able to do page by page PRs to update the CSS now.

### CODE REVIEW

**Comment by RoboticMind** [comment]
Looks like you need to update your index.htm because of #177

File: _includes/_head.htm [hunk 1]
```diff
@@ -8,13 +8,12 @@
 	<meta name="author" content="https://github.com/grctest/Gridcoin-Site/graphs/contributors">
 	<meta name="description" content="{{ page.description | escape }}">
 
-	<!--     Fonts and icons     -->
-	<link href="https://fonts.googleapis.com/css?family=Montserrat:400,700,200" rel="stylesheet" />
+	<!--     Fonts and icons     
+	<link href="https://fonts.googleapis.com/css?family=Montserrat:400,700,200" rel="stylesheet" />-->
```
Comment (barton2526) [inline _includes/_head.htm]: There is no reason given for this commenting out.  Is this part of the PR?

File: _layouts/layout.htm [hunk 2]
```diff
@@ -7,11 +7,9 @@
 	{% include _footer.htm %}
 
 	<!--   Core JS Files   -->
-	<script src="/assets/js/core/jquery.3.2.1.min.js" type="text/javascript"></script>
+	<script src="/assets/js/core/jquery-3.4.1.min.js" type="text/javascript"></script>
```
Comment (barton2526) [inline _layouts/layout.htm]: Note: JQuery is also being updated.  Does this affect/break anything?

Comment (ShmoogleOsukami) [reply _layouts/layout.htm]: As far as my local tests goes, Everything seems to function as before. Bootstrap isn't reporting any problems nor is NowUI. Everything looks correct.

File: _includes/_head.htm (hunk 1 above)
Comment (ShmoogleOsukami) [reply _includes/_head.htm]: After updating Bootstrap the font changed and messed with the layout of the site. Commenting this out fixed it to a degree.

I should really fix the Font once and for all but we have 3 things messing with it. Bootstrap does stuff to the font, NowUI messes with what bootstrap does and then we manually define a font as well. 

I'd sooner just use bootstraps default font or one of the fonts it has with it.

I don't know much about setting up fonts for websites. I should look into it.

Comment (ShmoogleOsukami) [reply _includes/_head.htm]: Hang on, I got a fix for this, I can reinstate the font, Just have to fix a few oddly sized bits of text on the index page.

Comment (ShmoogleOsukami) [reply _includes/_head.htm]: This is with the font,
screencapture-127-0-0-1-4000-2019-05-20-15_25_59

This is without the font. It looks better without but ideally, I'd like to remove Now-ui so bootstraps responsive font stuff stops getting overridden.

Comment (barton2526) [reply _includes/_head.htm]: Recommend a new PR to revisit the font issue separately.  Then I could just give the current PR the okay.  I think the font discussion should be a separate one.

**Comment by barton2526** [comment]
Approved

**Comment by barton2526** [comment]
Waiting for travis

**Comment by barton2526** [comment]
I would like 1 more review before we merge

File: index.htm [hunk 3]
```diff
@@ -368,11 +289,11 @@ <h4>Alternative Linux sources</h4>
             </div>
             <!--/GRC Research Downloads-->
             <!--MISC-->
-            <div class="col-xs-12 col-sm-6">
+            <div class="col-12 col-sm-6">
                 <h4>Other downloads</h4>
-                    <a class="btn btn-round grcbtn" href="https://boinc.berkeley.edu/download_all.php">BOINC Clients</a><br/>
-                    <a class="btn btn-round grcbtn" href="https://boinc.berkeley.edu/wiki/Add-on_software">BOINC Add-on software</a><br/>
-                    <a class="btn btn-round grcbtn" href="https://github.com/marius311/boinc-server-docker">BOINC server Docker container</a>
+                    <a class="btn rounded-pill grcbtn" href="https://boinc.berkeley.edu/download_all.php">BOINC Clients</a><br/>
```
Comment (cyrossignol) [inline index.htm]: The other merged `btn-round` download links came back with the NOW-UI class. Could use `rounded-pill` as well so the font weight matches--that class will go away anyway.

File: index.htm [hunk 4]
```diff
@@ -6,52 +6,52 @@
 <section class="intro" data-background-color="color">
     <div class="container">
         <div class="row">
-            <div class="col-xs-12 text-center">
-                <img src="/assets/img/GridcoinBlocks_Header_Transparent.png" alt="logo">
-                <h4 class="text-bold">Rewarding Volunteer Distributed Computing</h4>
+            <div class="col-12 text-center">
+                <img class="img-fluid" src="/assets/img/GridcoinBlocks_Header_Transparent.png" alt="Gridcoin Logo">
+                <h1 class="text-weight-bold">Rewarding Volunteer Distributed Computing</h1>
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply index.htm]: Opted for keeping it a H1 tag but styling as a H2 should help with the mobile visuals greatly.

**Comment by ShmoogleOsukami** [comment]
I've updated all the pill buttons to the correct class and went ahead and removed most of the CSS in gridcoin.css for the index page and replaced them with bootstrap classes.
Layout is largely the same just minor spacing differences.

File: index.htm [hunk 5]
```diff
@@ -6,52 +6,52 @@
 <section class="intro" data-background-color="color">
     <div class="container">
         <div class="row">
-            <div class="col-xs-12 text-center">
-                <img src="/assets/img/GridcoinBlocks_Header_Transparent.png" alt="logo">
-                <h4 class="text-bold">Rewarding Volunteer Distributed Computing</h4>
+            <div class="col-12 text-center">
+                <img class="img-fluid" src="/assets/img/GridcoinBlocks_Header_Transparent.png" alt="Gridcoin Logo">
+                <h1 class="h2 pt-3 text-weight-bold">Rewarding Volunteer Distributed Computing</h1>
```
Comment (cyrossignol) [inline index.htm]: Didn't you want `font-weight-bold` here?

File: index.htm (hunk 4 above)
[the remark this answers is not in the document]
Comment (cyrossignol) [reply index.htm]: Great 👍

File: index.htm (hunk 5 above)
Comment (ShmoogleOsukami) [reply index.htm]: oh nice catch yeah. Will fix.

Comment (cyrossignol) [reply index.htm]: Resolved.

**Pull request merged**